games-simulation/simutrans/simutrans-0.122.0.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop flag-o-matic xdg
+
+MY_PV=${PV/0./}
+MY_PV=${MY_PV//./-}
+SIMUPAK_64="simupak64-${MY_PV%-[0-9]*}-0.zip"
+SIMUPAK_128_V="2.8.1"
+SIMUPAK_128_BRITAIN="pak128.Britain.1.18-120-3.zip"
+SIMUPAK_128_GERMAN="PAK128.german_2.0_for_ST_122.0.zip"
+SIMUPAK_192_COMIC="pak192.comic.0.5.zip"
+
+DESCRIPTION="A free Transport Tycoon clone"
+HOMEPAGE="https://www.simutrans.com/"
+SRC_URI="mirror://sourceforge/simutrans/simutrans-src-${MY_PV}.zip
+	https://simutrans-germany.com/translator/data/tab/language_pack-Base+texts.zip -> simutrans_language_pack-Base+texts-${PV}.zip
+	mirror://sourceforge/simutrans/${SIMUPAK_64} -> simutrans_${SIMUPAK_64}
+	pak128? ( https://download.sourceforge.net/simutrans/pak128/pak128%20for%20ST%20120.4.1%20%28${SIMUPAK_128_V}%2C%20priority%20signals%20%2B%20bugfix%29/pak128.zip -> simutrans_pak128-${SIMUPAK_128_V}.zip )
+	pak128-britain? ( mirror://sourceforge/simutrans/${SIMUPAK_128_BRITAIN} -> simutrans_${SIMUPAK_128_BRITAIN} )
+	pak128-german? ( mirror://sourceforge/simutrans/${SIMUPAK_128_GERMAN} -> simutrans_${SIMUPAK_128_GERMAN} )
+	pak192-comic? (
+		mirror://sourceforge/simutrans/${SIMUPAK_192_COMIC} -> simutrans_${SIMUPAK_192_COMIC}
+		https://www.dropbox.com/s/3wwyrajrr2oqzo6/coalwagons.rar?dl=1 -> simutrans_coalwagonfix.rar
+	)"
+
+LICENSE="Artistic"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="freetype +pak128 +pak128-britain +pak128-german +pak192-comic upnp"
+
+RDEPEND="
+	sys-libs/zlib
+	app-arch/bzip2
+	media-libs/sdl-mixer[midi]
+	media-libs/libpng:0
+	media-libs/libsdl2[sound,video]
+	freetype? ( media-libs/freetype )
+	upnp? ( net-libs/miniupnpc )
+	app-arch/zstd
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	|| ( app-arch/zip app-arch/unzip )
+	virtual/imagemagick-tools[png]
+	|| ( app-arch/rar app-arch/unrar )
+"
+
+S=${WORKDIR}
+
+PATCHES=("${FILESDIR}/simutrans-0.122.0-Fix-Makefile.patch" )
```

Comment:
The spacing here is weird. Either don't use spaces at all, or place it both after `(` and before `)`.